source/parser.hera

```diff
@@ -2047,7 +2048,8 @@ PropertyDefinition
       type: "SpreadProperty",
       children: [ws, dots, exp],
       names: exp.names,
-      value: [dots, exp],
+      dots,
+      value: exp,
```

Comment:
I'd name this `expression` instead of `value`.

Reply:
This is to match regular `Property` AST nodes.  In particular, this code relies on it when checking `if (part.value && !["CallExpression", "MemberExpression", "Identifier"].includes(part.value.type)) {` — this way, it's the same check for both `Property` and `SpreadProperty`.

But I agree that `SpreadProperty` is special, so perhaps this doesn't make sense.  Do you still think `expression` is best?

Reply:
In that case either is fine.